app/frontend/src/features/communities/discussion/DiscussionsListPage.tsx

```diff
@@ -0,0 +1,135 @@
+import { Dialog, DialogContent, makeStyles } from "@material-ui/core";
```

Comment:
This is intentionally copy/pasted from `DiscussionSection` for now. I'll likely refactor this in an upcoming part to share more markups between the two if that makes sense later on. But for now, duplicating seems like the cheapest way to distinguish the two so the bottom button can behave differently.